Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -595,6 +595,12 @@ - (void)viewDidDisappear:(BOOL)animated {
 
 - (void)traitCollectionDidChange:(nullable UITraitCollection *)previousTraitCollection {
     [super traitCollectionDidChange:previousTraitCollection];
+    NSArray<UICollectionViewCell *> *placeholderCells = self.placeholderCells.allValues;
+    //forces trait collection update on placeholder cells
+    for (UICollectionViewCell *cell in placeholderCells) {
+        [self.view insertSubview:cell atIndex:0];
+        [cell removeFromSuperview];
```

Comment:
`traitCollection` is nonnull - they could reset it to a bogus trait collection but we'd catch that by seeing the resulting layout issues